runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/process.json

```diff
@@ -330,6 +330,11 @@
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.FilePath.Info": "Odnosi się do ścieżki, gdzie punktem początkowym jest folder, w którym bot ląduje po zalogowaniu się do konta, np. nazwaFolderu/nazwaPliku.xlsx",
     "Process.Details.Modeler.Actions.SharePointExcel.OpenFileFromRoot.ListName.Info": "Domyślna nazwa to 'Documents'",
     "Process.Details.Modeler.Actions.CloudExcel.CloseSession.Label": "Zakończ sesję",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.Label": "Usuń kolumny",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.StartColumn": "Kolumna początkowa",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.EndColumn": "Kolumna końcowa",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.StartColumn.Info": "Nazwa kolumny początkowej (np. A)",
+    "Process.Details.Modeler.Actions.CloudExcel.DeleteColumns.EndColumn.Info": "Nazwa kolumny końcowej (np. H), jeśli nie określono wartości, usunięta zostanie tylko kolumna początkowa",
```

Comment:
desktop excel examples are in form ..., np. sth - no brackets, yes comma
+
grammar suggestion:
Nazwa kolumny końcowej, np. H. Jeżeli nie określono wartości, usunięta zostanie tylko kolumna początkowa